src/main/java/org/miracum/etl/fhirtoomop/mapper/QuestionnaireResponseMapper.java

```diff
@@ -56,6 +59,7 @@ public OmopModelWrapper map(QuestionnaireResponse srcQuestionnaireResponse, bool
             noFhirReferenceCounter.increment();
             return null;
         }
+        var appReasonList = srcQuestionnaireResponse.getItem().stream().filter(questionnaireResponseItemComponent -> questionnaireResponseItemComponent.getLinkId().equals("service-type")).toList();
```

Comment:
can we change variable name as `appointmentReasonList` to avoid confusion.
also define a constant for `service-type` and check whether we can process appointment reason thing only for schedule appointment questionnaire response
thanks

Reply:
done.